### PR TITLE
Bug/234

### LINKED ISSUE #234: Autopush should accept JSON formatted env args
Some args are compound and are normally expressed as JSON.  (e.g. AUTOEND_CRYPTO_KEYS, fcm creds, etc.)
Ops already has these as JSON, so we should make sure they're accepted.

### PR DESCRIPTION
**Pull request by jrconlin**
## Description
AUTH_KEYS were being rejected due to invalid characters. This was due to the fact that the while the sample config was not JSON Formatted, the ops key was. 

This is a stupid fix to ensure that keys are accepted.

## Testing

Tests included.

Closes #234, #233

### CODE REVIEW

**Comment by tublitzed** [comment]
Hey, JR I _think_ after poking around a bit last week that we might need to fix [this one](https://github.com/mozilla-services/autopush-rs/issues/233) before CI passes.

**Comment by jrconlin** [comment]
Ah! missed that one. Looks like I incorporated it into this PR.

File: autoendpoint/src/settings.rs [hunk 1]
```diff
@@ -106,15 +106,19 @@ impl Settings {
 
     /// Initialize the fernet encryption instance
     pub fn make_fernet(&self) -> MultiFernet {
-        let fernets = Self::read_list_from_str(&self.crypto_keys, "Invalid AUTOEND_CRYPTO_KEYS")
+        let keys = &self.crypto_keys.replace('"', "").replace(" ", "");
+        let fernets = Self::read_list_from_str(&keys, "Invalid AUTOEND_CRYPTO_KEYS")
```
Comment (Mcat12) [inline autoendpoint/src/settings.rs:110]: Definitely a simple fix, but just noting here that something like `serde_json::decode::<Vec<String>>` should work.

Comment (jrconlin) [reply autoendpoint/src/settings.rs:110]: Yeah, I considered that, but the example we had was not properly formatted JSON. For now, I figured I should apply that fix until I can dig into seeing if we allowed similar values in Python.

Comment (Mcat12) [reply autoendpoint/src/settings.rs:110]: Ah, ok so you want to allow the previous input as well :+1:

Comment (jrconlin) [reply autoendpoint/src/settings.rs:110]: Heh, well, if this is going to be a drop-in replacement, it needs to follow the same rules we had before.

Not super happy about it, but it's something we can address later if need be.

Comment (jrconlin) [reply autoendpoint/src/settings.rs:110]: Checked, and the autopush python [sample file](https://github.com/mozilla-services/autopush/blob/master/configs/autopush_endpoint.ini.sample#L28) also used non-JSON values.

**Review by pjenvey — APPROVED** [review]
It'd be nice to switch to strictly the JSON parsing later once ops switches over (and remove the extra parsing step from `auth_keys`), can we log an issue so we don't forget?

**Pull request merged**